spec.md

```diff
@@ -188,6 +188,114 @@ not implemented the Deploy section SHOULD be ignored and the Compose file MUST s
 
 `deploy` specifies the configuration for the deployment and lifecycle of services, as defined [here](deploy.md).
 
+### blkio_config
+
+`blkio_config` defines a set of configuration options to set block IO limits for this service.
+                         
+```yml
+services:
+  foo:
+    image: busybox
+    blkio_config:
+       weight: 300
+       weight_device:
+         - path: /dev/sda
+           weight: 400
+       device_read_bps:
+         - path: /dev/sdb
+           rate: '12mb'
```

Comment:
See my other comment; these settings are _per device_; ideally we would _flip_ the configuration to be grouped per device. These configs are also non-portable (devices will be different per environment, which makes it not very suitable for a compose-file that is to be shared between developers / environments)